src/pretix/static/pretixcontrol/js/ui/editor.js

```diff
@@ -397,6 +398,7 @@ var editor = {
         editor.fabric.on('object:added', editor._create_savepoint);
         editor.fabric.on('selection:cleared', editor._update_toolbox);
         editor.fabric.on('selection:created', editor._update_toolbox);
+        editor.fabric.on('selection:updated', editor._update_toolbox);
         editor.fabric.on('object:selected', editor._update_toolbox);
```

Comment:
From the [changelog](http://fabricjs.com/v2-breaking-changes-2) object:selected was deprecated in v2 and removed in v3. So we could (should?) remove it. I removed it and thanks to selection:created it still works.

```suggestion
```